### ISSUE TITLE
[Regression][Devel] After leaving airplane mode, pressing "reboot" makes unity crash

### ISSUE THREAD

**Issue by Flaburgan** [ubports/ubuntu-touch#1308]
- Device: Fairphone 2
- Channel: devel
- Build: 2019-11-25

### Steps to reproduce
Turn on and then off Airplane mode
From the top menu, press Shutdown
In the menu, press Restart

### Expected behavior
The phone should restart

### Actual behavior
Unity crashes and reload, without the phone rebooting.

### Logfiles and additional information
This is a regression since the merge of the new Mir and Unity in `devel`.

I don't find anything in `.cache/upstart/unity8.log`, is there other places to look?

**Comment by FlorianPaintner** [comment]
Behavior cannot be confirmed for channel stable.
Behavior cannot be confirmed for channel devel 1373 (april 10th 2020)
Device reboots propperly

**Comment by Flaburgan** [comment]
I still reproduce it, even if I now use the Halium 7.1 beta port for FP2. I will make a video of the bug.

**Comment by UniversalSuperBox** [comment]
I don't believe this is related to flight mode, but rather the number of apps open. Indeed, Lomiri appears to be crashing before it can issue a shutdown command to Upstart
